smart/plugin.go

```diff
@@ -166,6 +177,10 @@ func (sc *SmartCollector) GetMetricTypes(_ plugin.ConfigType) ([]plugin.MetricTy
 
 //GetConfigPolicy returns a ConfigPolicy
 func (p *SmartCollector) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	c := cpolicy.New()
-	return c, nil
+	cp := cpolicy.New()
+	rule, _ := cpolicy.NewStringRule("proc_path", false, "/proc")
```

Comment:
This alternate mount point is not used anywhere; it needs to be passed to code listing partitions: https://github.com/intelsdi-x/snap-plugin-collector-smart/blob/master/smart/smart.go#L266 .